predictoor/pdr-faq.md

```diff
@@ -0,0 +1,140 @@
+<details>
+
+<summary>How does Predictoor compare to prediction markets? Like Predictoor, they use $ for accountability.</summary>
+
+Prediction markets are for one-off predictions like “will Biden win in 2024?” whereas Predictoor-style prediction feeds are for a continuous stream of predictions.
+
+</details>
+
+<details>
+
+<summary>How does Predictoor compare to Chainlink? Like Predictoor, it has price feeds on-chain.</summary>
+
+Chainlink provides current prices on-chain, and Predictoor makes predictions for prices in the future.
+
+</details>
+
+<details>
+
+<summary>How does Predictoor compare to Numerai? Like Predictoor, it crowd-sources predictions, related to trading.</summary>
+
+They are different:  
+- Numerai takes predictions, then aggregates them and uses them to trade directly. It operates as a hedge fund.  
+- Whereas Predictoor takes predictions, then aggregates them and sells the aggregate. It operates as data feeds.  
+
+Secondary differences:  
+- Numerai focus is tradfi trading, as a hedge fund.  
+- Whereas Predictoor is pure datafeeds, and applies to any vertical. Its first use case is defi trading.  
+- Currently Numerai is mostly centralized.  
+- Whereas Predictoor is decentralized. Predictoor tech may be useful to Numerai to help decentralize  
+
+</details>
+
+<details>
+
+<summary>How does Predictoor compare to Numerai Signals? People supplying to Numerai Signals don’t get paid out from trading (like the main Numerai product).</summary>
+
+In Numerai Signals, payout is a function of the submitter’s “originality” compared to other signals. These signals are used as inputs for prediction models in Numerai main product.  
+
+Whereas in Predictoor, payout is a function of the the accuracy of the submitter’s prediction, compared to the true value. These signals are predictions, directly.
+
+</details>
+
+### For Predictoors
+
+<details>
+
+<summary>What happens if your prediction is good or bad?</summary> 
+
+If the prediction is wrong, your staked amount on that prediction is going to be slashed and divided between people who submitted the right predictions.
+
+</details>
+
+<details>
+
+<summary>Is it possible to predict without using bots?</summary>
+
+Before we answer, please understand that to make decent money, it will take hundreds or thousands of submissions, guided by AI/ML models. So doing this manually would be tedious and error-prone. Therefore a bot is the most practical way. This is why OPF has made it easy to run a bot. Here are the Readme of both [Predictoor](https://github.com/oceanprotocol/pdr-backend/blob/main/READMEs/predictoor.md) and [Trader](https://github.com/oceanprotocol/pdr-backend/blob/main/READMEs/trader.md).  
+
+To the question "Is it possible to predict without a bot", the specific answer is "Yes". Here are the details
+- Somehow you need to submit an up/down prediction, with OCEAN stake, as a transaction to Sapphire chain. Here are the Possible ways to do such a transaction:
```

Comment:
Remove `Here are the details`, line-break, and the bullet in front of `Somehow you need`. Why: clearer.

Add a paragraph break before `Here are Possible ways`. Why: clearer.

`Possible` -> `possible`. Why: grammar.